client/src/pages/Messages.tsx

```diff
@@ -277,13 +279,43 @@ const Profile = () => {
   const [messages, setMessages] = useState<WebSocketMessage[]>([])
   const { sendMessage, lastMessage } = useContext(WebSocketContext)
   const { user } = useUser()
+  const { session } = useSession()
+  const groupId = '1980b837-6506-4b53-8363-f8d21c43822e'
```

Comment:
we should pull this info from the database. Meaning we should ask the DB "What groups am I in?" and then allow the end user to select those groups and send messages to those groups instead of it being hardcoded like it is right now